src/xcode/ENA/ENA/Source/Services/Risk/Risk.swift

```diff
@@ -54,16 +54,61 @@ extension Risk {
 			Analytics.collect(.testResultMetadata(.setDateOfConversionToCheckinHighRisk(Date())))
 		}
 
-		let tracingRiskLevelPerDate = enfRiskCalculationResult.riskLevelPerDate
-		let checkinRiskLevelPerDate = checkinCalculationResult.riskLevelPerDate
+		let totalRiskLevel = Self.totalRiskLevel(
+			enfRiskCalculationResult: enfRiskCalculationResult,
+			checkinCalculationResult: checkinCalculationResult
+		)
+		Log.debug("[Risk] totalRiskLevel: \(totalRiskLevel)", log: .riskDetection)
+		
+		var previousTotalRiskLevel: RiskLevel = .low
+
+		if let previousENFRiskLevel = previousENFRiskCalculationResult,
+		   let previousCheckinRiskLevel = previousCheckinCalculationResult {
+
+			previousTotalRiskLevel = Self.totalRiskLevel(
+				enfRiskCalculationResult: previousENFRiskLevel,
+				checkinCalculationResult: previousCheckinRiskLevel
+			)
+		} else if let previousENFRiskLevel = previousENFRiskCalculationResult {
+			previousTotalRiskLevel = previousENFRiskLevel.riskLevel
+		} else if let previousCheckinRiskLevel = previousCheckinCalculationResult {
+			previousTotalRiskLevel = previousCheckinRiskLevel.riskLevel
+		}
+		
+		Log.debug("[Risk] previousTotalRiskLevel: \(previousTotalRiskLevel)", log: .riskDetection)
+
+		let riskLevelChange: RiskLevelChange
+		if previousTotalRiskLevel == totalRiskLevel {
+			riskLevelChange = .unchanged(totalRiskLevel)
+		} else {
+			riskLevelChange = previousTotalRiskLevel > totalRiskLevel ? .decreased : .increased
+		}
+		
+		Log.debug("[Risk] riskLevelChange: \(riskLevelChange)", log: .riskDetection)
 
-		Log.debug("[Risk] tracingRiskLevelPerDate: \(tracingRiskLevelPerDate)", log: .riskDetection)
-		Log.debug("[Risk] checkinRiskLevelPerDate: \(checkinRiskLevelPerDate)", log: .riskDetection)
```

Comment:
Would it make sense to still log these somewhere, in case we need to debug later?

Reply:
Good point. Added logs.